equinox/serialisation.py

```diff
@@ -218,11 +217,17 @@ def tree_deserialise_leaves(
         import equinox as eqx
         import jax.random as jr
 
-        model = eqx.nn.MLP(2, 2, 2, 2, key=jr.PRNGKey(0))
-        eqx.tree_serialise_leaves("some_filename.eqx", model)
-        model2 = eqx.tree_deserialise_leaves("some_filename.eqx", model)
-        ```
+        model_original = eqx.nn.MLP(2, 2, 2, 2, key=jr.PRNGKey(0))
+        eqx.tree_serialise_leaves("some_filename.eqx", model_original)
+        model_loaded = eqx.tree_deserialise_leaves("some_filename.eqx", model_original)
 
+        # (Recommended) To partially load weights
+        model_partial = eqx.tree_at(lambda mlp: mlp.layers[-1], model_loaded, model_original)
```

Comment:
I like this example.

Reply:
Should I remove the alternative one? Just wanted to somehow capture the DFS sub-structure bit for partial loading.

Reply:
yep